client/asset/interface.go

```diff
@@ -10,6 +10,38 @@ import (
 	"decred.org/dcrdex/dex"
 )
 
+// WalletTrait is a bitset indicating various optional wallet features, such as
+// the presence of auxiliary methods like Rescan and Send.
+type WalletTrait uint64
+
+const (
+	WalletTraitRescanner     WalletTrait = 1 << iota // The Wallet is an asset.Rescanner.
+	WalletTraitSender                                // The Wallet is an asset.Sender.
```

Comment:
Why do we need the sender trait? I would think every wallet would have the send functionality by default, and the additional information we would need to know is whether or not the wallet supports sweeping.

Reply:
We can certainly make `Send` standard, part of `asset.Wallet`